apps/web/content/blog/authors.ts

```diff
@@ -7,7 +7,10 @@ export type Author = {
   };
 };
 
-export const authors = {
```

Comment:
I don't mind, but why?

Reply:
This satisfied types in external files when using authors[string] when using satisfied it complains